torch/csrc/jit/interned_strings.h

```diff
@@ -8,6 +8,15 @@
 
 namespace torch { namespace jit {
 
+enum class SymbolNamespace : char {
+  ONNXOperator = 'o',
+  JITOperator = 'j',
+  ATenOperator = 't',
+  // NB: ONNX and ATen attributes all live in a global namespace, as
+  // their interpretation depends on the operator name (which is namespaced)
+  Attribute = 'a',
+};
```

Comment:
Is this even used?

Reply:
Oops, not yet, this is from the future :)